rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -995,6 +995,41 @@ public void onSuccess(String result) {
         }
     }
 
+    public void undeployNodeSource() {
+        if (model.getSelectedNodeSource() != null) {
+            String nodeSourceName = model.getSelectedNodeSource().getSourceName();
+
+            if (model.getSelectedNodeSource() != null) {
+                confirmUndeployNodeSource("Confirm undeployment of <strong>" + "NodeSource " + nodeSourceName +
+                                          "</strong>", new NodeRemovalCallback() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "</strong>" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)